packages/core/integration-tests/test/scope-hoisting.js

```diff
@@ -3574,6 +3574,27 @@ describe('scope hoisting', function () {
   });
 
   describe('commonjs', function () {
+    it('should wrap when this could refer to an export', async function () {
```

Comment:
Please also move the test case to `/integration/scope-hoisting/commonjs/...`

And it looks like there is already a similar case `exports-self-reference`, so maybe call this one similarly. Though this isn't really a good system to make it easier for humans to find test cases again...